test/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ list( APPEND test_input
   testinput/smap_sss_rss.nc
   testinput/hgodas_insitu.nc
   testinput/hgodas_sst.nc
+  testinput/SG547_0001_dn_AOML.nc
```

Comment:
can you rename this to something more meaningful? i have no idea this is glider data given the filename. Ideally have input and output filenames the same.